build-tools/src/main/groovy/org/jetbrains/kotlin/KonanTest.groovy

```diff
@@ -542,6 +544,11 @@ fun runTest() {
             return false
         }
 
+        def apiVersion = findLinesWithPrefixesRemoved(text, '// !API_VERSION: ')
+        if (apiVersion.size() != 0 && !apiVersion.contains("1.4")) {
```

Comment:
Shouldn't we update the same check for language version above?

Reply:
Maybe rewrite that logic in Kotlin finally?

Reply:
I'm going to rewrite this and there is something already done in this way in another branch. But this fix is just to fix tests failures now.

As for language_version, I don't think this is necessary as tests pass.

Reply:
> As for language_version, I don't think this is necessary as tests pass.

But currently we exclude all tests with LANGUAGE_VERSION=1.4, don't we?